src/common/lib/api.ts

```diff
@@ -38,6 +39,7 @@ export default class Api {
       timeout: config.timeout || 20000,
       logging: config.logging || false,
       logger: config.logger || console.log,
+      network: config.network,
```

Comment:
we should probably give a default value here? like `"arweave.N.1"` - is that right?

Reply:
Default value is undefined. Do not send any header if not specified